bluebottle/members/serializers.py

```diff
@@ -194,7 +195,7 @@ def validate_email_confirmation(self, attrs, source):
     class Meta:
         model = BB_USER_MODEL
         fields = ('id', 'username', 'first_name', 'last_name',
-                  'email', 'password', 'jwt_token')
+                  'email', 'password', 'jwt_token', 'primary_language')
```

Comment:
Yeah, the frontend determines the primary_language depending on what language the using is currently using. It posts this to the backend. (And without it we default to the  LANGUAGE_CODE in properties.py)